src/BLL/Concrete/EmailSender.cs

```diff
@@ -0,0 +1,63 @@
+﻿#region
+using System.Configuration;
+using System.Net;
+using System.Net.Mail;
+using System.Net.Mime;
+using System.Text;
+using System.Threading.Tasks;
+using BLL.Abstract;
+using Domain;
+
+#endregion
+
+namespace BLL.Concrete
+{
+    public class EmailSender : IMailSender
+    {
+        public async Task<bool> SendMailAsync(string subject, string body, string emailTo)
+        {
+            EmailCredential credential = GetCredentialsFromConfig();
+            
+            try
+            {
+                using (SmtpClient client = new SmtpClient(credential.Host, credential.Port)
+                {
+                    EnableSsl = credential.UseSsl,
+                    UseDefaultCredentials = false,
+                    DeliveryMethod = SmtpDeliveryMethod.Network,
+                    Credentials = new NetworkCredential(credential.Email, credential.Password),
+                    Timeout = 2000
+                })
+                using (MailMessage mailToClient = new MailMessage(credential.Email, emailTo)
+                {
+                    IsBodyHtml = true,
+                    Body = body,
+                    Subject = subject
+                })
+                {
+                    await client.SendMailAsync(mailToClient);
+                }
+            }
+            catch (SmtpException)
+            {
+                return false;
+            }
+            return true;
+        }
+
+        private EmailCredential GetCredentialsFromConfig()
+        {
+            EmailCredential credential = new EmailCredential
+            {
+                Email = "rp16.serious.games@gmail.com",
```

Comment:
@GasperPaul @yehorchankov  Should we leave hardcoded or should it be in some `.config` file? 
Or leave it as it is for now and then change if needed?